ooniprobe/View/Informed consent/ICSwipeViewController.h

```diff
@@ -0,0 +1,22 @@
+#import <UIKit/UIKit.h>
+#import "Onboarding1ViewController.h"
+#import "Onboarding2ViewController.h"
+#import "Onboarding3ViewController.h"
+#import "Onboarding4ViewController.h"
+
+@interface ICSwipeViewController : UIViewController <UIPageViewControllerDataSource, UIGestureRecognizerDelegate> {
+    NSArray *viewControllers;
+    int question_number;
```

Comment:
I maybe am confused, but it seems to me this variable is only set to `1` but otherwise it's not used.